framework/audio/engine/iaudioplayer.h

```diff
@@ -43,7 +43,10 @@ class IAudioPlayer : MODULE_EXPORT_INTERFACE
 public:
```

Comment:
I thought you prefer to keep all the types at "*modulename*types" (PlayStatus)

Reply:
Yes, fixed :)